qa-tests-backend/src/test/groovy/UpgradesTest.groovy

```diff
@@ -189,15 +189,20 @@ class UpgradesTest extends BaseSpecification {
         given:
         "Default policies in code"
 
+        // @TODO(ROX-10249): Policy is no longer guarded by a feature flag
```

Comment:
[nit] Move this in line with the id and change to "Remove when policy is no longer guarded by a feature flag"

Reply:
Done! Thanks